src/JJMasterData.Core/WebComponents/JJFormView.cs

```diff
@@ -466,7 +466,7 @@ private HtmlBuilder GetHtmlElementList(InsertAction action)
         sHtml.AppendHiddenInput($"current_painelaction_{Name}", "ELEMENTLIST");
         sHtml.AppendHiddenInput($"current_selaction_{Name}", "");
 
-        var dicParser = new DataDictionaryDao(EntityRepository).GetMetadata(action.ElementNameToSelect);
+        var dicParser = new DataDictionaryDAO(EntityRepository).GetMetadata(action.ElementNameToSelect);
```

Comment:
DataDictionaryDao should not be hard-coded right? The correct would be `JJServiceCore.DataDictionaryRepository.`